app/views/movies/index.html.erb

```diff
@@ -0,0 +1,179 @@
+<div class="row mb-3">
+  <div class="col-md-12">
+    <h1>
+      List of all movies
+    </h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="row mb-3">
+  <div class="col-md-8 offset-md-2">
+    <h2>
+      Add a new movie
+    </h2>
+
+    <form action="/insert_movie" method="post">
+      <div class="form-group">
+        <label for="title">
+          Title
+        </label>
+
+        <input type="text" id="title" name="input_title" class="form-control">
+      </div>
+
+      <div class="form-group">
+        <label for="year">
+          Year
+        </label>
+
+        <input type="text" id="year" name="input_year" class="form-control">
+      </div>
+
+      <div class="form-group">
+        <label for="description">
+          Description
+        </label>
+
+        <textarea id="description" name="input_description" class="form-control" rows="3"></textarea>
+      </div>
+
+      <div class="custom-control custom-checkbox mb-3">
+        <input type="checkbox" class="custom-control-input" id="released" name="input_released" value="1">
+
+        <label class="custom-control-label" for="released">Released</label>
+      </div>
```

Comment:
Is there a missing newline after this checkbox's `div.form-group`?

Reply:
@raghubetina Are you referring to the `div.custum-control` on line 42? Yes, that is missing a newline. 👍